.editorconfig

```diff
@@ -0,0 +1,1008 @@
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = false
+max_line_length = 140
+tab_width = 4
+ij_continuation_indent_size = 8
+ij_formatter_off_tag = @formatter:off
+ij_formatter_on_tag = @formatter:on
+ij_formatter_tags_enabled = false
+ij_smart_tabs = false
+ij_visual_guides = none
+ij_wrap_on_typing = true
+
+[*.css]
```

Comment:
Let's remove this one too as we never edit the css files.

Reply:
We also have `*.js` files, and we probably don't edit them. I will remove front-end related file extensions, even if we have such files in the repo.